.github/workflows/release_pre_built/action.yml

```diff
@@ -0,0 +1,51 @@
+name: "Release pre built"
+description: "Builds elixir release, ExDoc and generates docs"
+inputs:
+  otp:
+    description: "The major OTP version"
+  otp_version:
+    description: "The minor OTP version"
```

Comment:
I think the naming is a bit confusing here because in practice this would be e.g.:

```
otp: 25
otp_version: 25.2.1
```

while the former is the major version the latter isn't the _minor_ version, it's _the_ version so to speak.

How about these names instead? If they make sense, can you send a patch?

```yaml
   otp_release:
     description: "The OTP release"
   otp_version:
     description: "The OTP version"
```

Reply:
Good find. I think we can keep the current names but change the descriptions, I will change it. :)

Reply:
Love it!